### PR TITLE
edit 3-1

### PR DESCRIPTION
**Pull request by timousT2**
please check it

### CODE REVIEW

**Comment by timousT** [comment]
what problem

**Comment by timousT2** [comment]
you type something wrong

File: test6-2.txt [hunk 1]
```diff
@@ -1,3 +1,7 @@
 edit 1
 
-edit 2
\ No newline at end of file
+edit 2
+
+edit 3
```
Comment (timousT) [inline test6-2.txt:5]: Am I type it worg?

Comment (timousT2) [inline test6-2.txt:5]: no. the wrong one is in line 7

File: test6-2.txt [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+
+edit 3-1
```
Comment (timousT2) [inline test6-2.txt:7]: no. the wrong one is in line 7

**Comment by timousT** [comment]
ok i will fix it.

**Pull request merged**